Dockerfile

```diff
@@ -1,4 +1,5 @@
 #VERSION 1.0.0
+FROM db-ex-common-sshproxy AS sshproxy
```

Comment:
I'm not sure this will work as per travis build error

Reply:
Travis was down, so I didn't bother to wait for it, sorry about that. I'll fix it. If you just run DC build, it correctly builds sshproxy first.

Reply:
aha, sorry!  I didn't realise